documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,53 +1,26 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} automatically creates a Prometheus `ServiceMonitor` that scrapes metrics from your {brandname} cluster.
 
-.Prerequisites
+.Procedure
 
-* Have an `oc` client.
-* Enable monitoring for user-defined projects on {ocp}.
+Enable monitoring for user-defined projects on {ocp} so that the `ServiceMonitor` resource is available on the {openshiftshort} cluster.
```

Comment:
I believe that `ServiceMonitor` resource is always available on OCP clusters. But if user workload monitoring isn't enabled it's instance presence won't have any effect.

Reply:
makes sense. I'll change.